CONTRIBUTING.md

```diff
@@ -217,7 +217,7 @@ sudo apt install socat
 1. Install BinderHub and its development requirements:
 
     ```bash
-    python3 -m pip install -e . -r dev-requirements.txt
+    python3 -m pip install -e . -r dev-requirements.txt -r helm-chart/images/binderhub/requirements.txt
```

Comment:
It should be possible to nest requirements files, try adding
```
-r helm-chart/images/binderhub/requirements.txt
```
to the end of `dev-requirements.txt`?

Reply:
@manics I did it, but couldn't decide if I should update [.travis.yaml](https://github.com/jupyterhub/binderhub/blob/0b5080ade21dbda5ad159e675bacf25c61d8e422/.travis.yml#L19-L22) (by removing line 22 and replacing line 19 with `pip install --upgrade . -r dev-requirements.txt`) or leave it as it is.